<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileExistsDialog</class>
 <widget class="QDialog" name="FileExistsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>82</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Exists</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="fileExistsLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="skipButton">
         <property name="text">
          <string>skip</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="skipAllButton">
         <property name="text">
          <string>skip all</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="overwriteButton">
         <property name="text">
          <string>overwrite</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="overwriteAllButton">
         <property name="text">
          <string>overwrite all</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
